src/components/Footer.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import Image from "next/image";
+import Container from "./commons/Container";
+
+const MenuItems = [
+  {
+    category: "Ecosystem",
+    childLink: [
+      {
+        imagePath: "/menu/dmc-explorer.svg",
+        testId: "DeFiMetaChain",
+        label: "DeFiMetaChain",
+        href: "https://defimetachain.org/",
+      },
+      {
+        imagePath: "/menu/defichain-com.svg",
+        testId: "DeFiChain.com",
+        label: "DeFiChain.com",
+        href: "https://defichain.com",
+      },
+      {
+        imagePath: "/menu/VerifiedContracts.svg",
+        testId: "Verified Contracts",
+        label: "Verified Contracts",
+        href: "",
+      },
+    ],
+  },
+  {
+    category: "Community",
+    childLink: [
+      {
+        imagePath: "/menu/twitter.svg",
+        testId: "Twitter",
+        label: "Twitter",
+        href: "https://twitter.com/defichain",
+      },
+      {
+        imagePath: "/menu/telegram.svg",
+        testId: "Telegram",
+        label: "Telegram",
+        href: "https://t.me/defiblockchain",
+      },
+    ],
+  },
+  {
+    category: "Blog",
+    href: "",
+    childLink: [
+      {
+        imagePath: "/menu/whitepaper.svg",
+        testId: "Whitepaper",
+        label: "Whitepaper",
+        href: "",
+      },
+      {
+        imagePath: "/menu/github.svg",
+        testId: "GitHub",
+        label: "GitHub",
+        href: "https://github.com/DeFiCh/metachain",
+      },
+    ],
+  },
+];
+
+function Footer() {
+  return (
+    <footer className="w-full relative pb-[40px] lg:pb-[70px] md:pb-[52px] sm:pb-[57.05] lg:pt-[104px] md:pt-[66px] pt-[79px] bottom-0 right-0">
+      <Container className="px-4 sm:px-10 md:px-10">
+        <div className="black-gradient-1-shadow backdrop-blur-[6px] bg-opacity-70 bg-black-900 flex flex-col sm:flex-row py-10 px-5 md:px-10 lg:px-[86px] rounded-[30px] black-gradient-1 gap-y-10 md:gap-0">
```

Comment:
check vertical padding in mobile view

Reply:
![image](https://user-images.githubusercontent.com/40191153/200723601-2e12c46c-7a60-406a-9f94-c2cc9f37245c.png)
i meant this